src/components/InputDeviceConfig/AudioVideo.vue

```diff
@@ -117,13 +138,27 @@ export default {
         this.$modV.setupMedia({ videoId: value });
         this.switchingVideo = false;
       }
+    },
+
+    maxGain() {
+      return this.$modV.gainNode.gain.maxValue.toPrecision(1 + 2).split("e")[0];
```

Comment:
We split off the scientific notation here as it kind of breaks the range slider.